README.md

```diff
@@ -2,31 +2,305 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Define Object-Relational Mapping
+- Save an `Employee` class instance to a database table.
+- Create an `Employee` class instance from the data in a table row.
+- Use the `PreparedStatement` interface to efficiently insert multiple `Employee` objects into a table.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+**Object Relational Mapping (ORM)** is a programming technique for converting
+between a relational data model and an object-oriented data model.
 
-## Topic 1
+In a subsequent lesson we will work with the **Jakarta Persistence API (JPA)**,
+which is a standard specification that defines how an ORM should interact with a relational database.
```

Comment:
In the pull request for the "What is JPA and Hibernate?" lesson, you specifically removed the word "relational". I assume that is because we can now use JPA with NoSQL databases, like MongoDB. For consistency, maybe remove the word "relational" here.

Reply:
ok, removed "relational".